README.md

```diff
@@ -3,9 +3,35 @@
 
 # OpenQA Public cloud Helper
 
-PublicCloud-Watcher or `pcw` is a web app which monitors, displays and deletes resources on various Cloud Service Providers (CSPs). It identifies abandoned instances by searching for certain tags and performs cleanup tasks. It also deletes old custom uploaded images. The behavior differs per CSP.
-
-The fastest way to run PublicCloud-Watcher is via the provided containers, as described in the [Running a container](#running-a-container) section.
+PublicCloud-Watcher (PCW) is a web app which monitors, displays and deletes resources on various Cloud Service Providers (CSPs).
+PCW has two main flows :
+
+1. Update run ( implemented in [ocw/lib/db.py](ocw/lib/db.py) )
```

Comment:
```suggestion
1. Periodic update runs (implemented in [ocw/lib/db.py](ocw/lib/db.py))
```

Reply:
I changed my mind :) see details https://github.com/SUSE/pcw/pull/152#discussion_r918938877

Reply:
> I would still add the word `periodic` to highlight that this is a recurring task.

I addressed that but not exactly like you asked. Check latest version and tell me if it is fine